octopoes/octopoes/core/service.py

```diff
@@ -395,6 +395,17 @@ def recalculate_scan_profiles(self, valid_time: datetime) -> None:
         )
         logger.info("Recalculated scan profiles")
 
+    def nibblet_toggle_phantom_result(self, origin_id: str, valid_time: datetime) -> None:
```

Comment:
This is never invoked as far as I can see? Or is the PR hence still in draft?

Reply:
Yes :)